visidata/loaders/html.py

```diff
@@ -104,7 +105,8 @@ def save_html(vd, p, *vsheets):
     with open(p, 'w', encoding='ascii', errors='xmlcharrefreplace') as fp:
         for sheet in vsheets:
 
-            fp.write('<h2 class="sheetname">%s</h2>\n'.format(sheetname=html.escape(sheet.name)))
+            if options.include_html_title:
```

Comment:
What if this became:
```
hdr = html.escape(options.html_title.format(sheet=self, vd=vd))
fp.write('<h2>'+hdr+'</h2>')
```

Then the default would be an empty string (empty h2 seems fine to me), and you could set it to `{sheet.name}`, and someone else could set it to `"TPS Report"`.